external/fv3fit/tests/training/test_train.py

```diff
@@ -126,23 +146,28 @@ def _get_dataset_precipitative(sample_func):
     )
 
 
-def _get_dataset_default(sample_func, data_2d_ceof=1):
+def _get_dataset_default(sample_func, data_2d_ceof=1, labeled=True):
     input_variables = ["var_in_2d", "var_in_3d"]  # 2d var will be clipped below
-    output_variables = ["var_out"]
     input_values = list(sample_func() for _ in input_variables)
     i_2d_input = input_variables.index("var_in_2d")
     input_values[i_2d_input] = input_values[i_2d_input].isel(z=0) * data_2d_ceof
-    i_3d_input = input_variables.index("var_in_3d")
-    output_values = [input_values[i_3d_input]]
+    if labeled:
+        output_variables = ["var_out"]
+        i_3d_input = input_variables.index("var_in_3d")
+        output_values = [input_values[i_3d_input]]
+        processed_dataset = _process(input_variables, input_values, output_variables, output_values)
+    else:
+        output_variables = []
+        processed_dataset = _process_unlabeled(input_variables, input_values)
```

Comment:
What breaks if the new model is given the original dataset default with the output variables? 
I think typically our models should be able to fit/predict on any dataset that has additional variables on top of the required inputs/outputs.

Reply:
Just eliminated this difference! You're right that we can just ignore the output variables.